src/components/form/ErrorLabel.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import './ErrorLabel.css'
 
+function reload(){
+  window.location.reload();
+}
+
 const ErrorLabel = ({ errorMessage }) => (
   <div className="FormEdit">
     <p>
       <i className="fas fa-exclamation-triangle" aria-hidden="true" />&nbsp;
       <span>{errorMessage}</span>
     </p>
-    <p onClick={() => {window.location.reload()}}> Click here to reload the page </p>
+    <p onClick={reload}> Click here to reload the page </p>
```

Comment:
Actually my comment was just that `window.location.reload` could be passed directly, no need to wrap in another function (anonymous or otherwise).
Or, was there some reason that doesn't work?